src/ngraph_builder.cc

```diff
@@ -1550,6 +1550,63 @@ static Status TranslateMatMulOp(
   return Status::OK();
 }
 
+static Status TranslateMaxOp(const Node* op,
+                             const std::vector<const Tensor*>& static_input_map,
+                             Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input, ng_max_op;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input, &ng_max_op));
+
+  bool tf_keep_dims;
+  if (GetNodeAttr(op->attrs(), "keep_dims", &tf_keep_dims) != Status::OK()) {
+    tf_keep_dims = false;
+  }
+
+  std::vector<int64> max_axes;
+  TF_RETURN_IF_ERROR(GetStaticInputVector(op, 1, static_input_map, &max_axes));
+
```

Comment:
We can add a check here to make sure the max_axes dimensions are within [-rank(input), rank(input))

Reply:
The testing framework throws an error if I try to give a value that is out of bounds. So when I added the check, I have no way of testing it.

Reply:
Added the check anyway